### DISCUSSION TITLE
Did the colors of content boxes change?

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#168] [Q&A] [answered]
I just updated to latest master on jupyter-book, sphinx-book-theme and pydata-theme, and the colors of the boxes are... not the ones they used to be:
- "tip" went from yellow to green-ish
- "warning" went from red to yellow

When did this happen and why? I can't find the PR which changed that. I'd like to go back to the old ones :sweat_smile:

**Comment by choldgraf** [comment, marked as answer]
ahh yes - this is because the underlying `pydata-sphinx-theme` changed those colors recently. I think that we should revert to the old colors in the sphinx-book-theme `master` branch before we cut new releases of Jupyter Book 👍

**Reply by choldgraf** [reply]
@fmaussion think you could open up an issue for this in the sphinx-book-theme? I think the changes we'd need to make would relate to these instructions:

https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/customizing.html#theme-variables

(a PR is also most-welcome if you have the interest!)
